packages/core/src/memory/index.ts

```diff
@@ -316,9 +316,10 @@ new Memory({
     metadata?: Record<string, unknown>;
     memoryConfig?: MemoryConfig;
   }): Promise<StorageThreadType> {
+    const id = threadId || this.generateId();
     const thread: StorageThreadType = {
-      id: threadId || this.generateId(),
```

Comment:
We have a genTitle api in Agent, should we move that here? ?

Reply:
oo nice I'll take a look thanks

Reply:
Looks like it generates a title based on the prompt but in this case the dev is calling createThread directly - I guess I could also just make title non null in createThread 🤔

Reply:
Or have a fallback like "New thread"